docs/manual/debugging.md

```diff
@@ -0,0 +1,133 @@
+# Debugging
+
+## Kubectl FDB Plugin
+
+You can use the [kubectl-fdb](/tree/master/kubectl-fdb) plugin when investigating issues and running imperative commands.
+
+If a cluster is stuck in a reconciliation you can use the `kubectl-fdb` plugin to analyze the issue:
+
+```
+$ kubectl fdb analyze example-cluster
+Checking cluster: default/example-cluster
+✔ Cluster is available
+✔ Cluster is fully replicated
+✖ Cluster is not reconciled
+✖ ProcessGroup: stateless-5 has the following condition: MissingProcesses since 2021-04-27 02:57:25 +0100 BST
+✖ ProcessGroup: storage-2 has the following condition: MissingProcesses since 2021-03-25 07:31:08 +0000 GMT
+✖ ProcessGroup: storage-2 has the following condition: PodFailing since 2021-03-25 18:23:22 +0000 GMT
+✖ ProcessGroup: storage-2 has the following condition: IncorrectConfigMap since 2021-03-26 19:20:56 +0000 GMT
+✖ ProcessGroup: transaction-5 has the following condition: MissingProcesses since 2021-04-27 02:57:24 +0100 BST
+✖ Pod default/example-cluster-storage-2 has unexpected Phase Pending with Reason:
+✖ Pod default/example-cluster-storage-2 has an unready container: foundationdb
+✖ Pod default/example-cluster-storage-2 has an unready container: foundationdb-kubernetes-sidecar
+✖ Pod default/example-cluster-storage-2 has an unready container: trace-log-forwarder
+Error:
+found issues for cluster example-cluster. Please check them
+```
+
+The plugin can also resolve these issues automatically:
+
+```
+$ kubectl fdb analyze example-cluster --auto-fix
+Checking cluster: default/example-cluster
+✔ Cluster is available
+✔ Cluster is fully replicated
+✖ Cluster is not reconciled
+✖ ProcessGroup: stateless-5 has the following condition: MissingProcesses since 2021-04-27 02:57:25 +0100 BST
+✖ ProcessGroup: storage-2 has the following condition: MissingProcesses since 2021-03-25 07:31:08 +0000 GMT
+✖ ProcessGroup: storage-2 has the following condition: PodFailing since 2021-03-25 18:23:22 +0000 GMT
+✖ ProcessGroup: storage-2 has the following condition: IncorrectConfigMap since 2021-03-26 19:20:56 +0000 GMT
+✖ ProcessGroup: transaction-5 has the following condition: MissingProcesses since 2021-04-27 02:57:24 +0100 BST
+✖ Pod default/example-cluster-storage-2 has unexpected Phase Pending with Reason:
+✖ Pod default/example-cluster-storage-2 has an unready container: foundationdb
+✖ Pod default/example-cluster-storage-2 has an unready container: foundationdb-kubernetes-sidecar
+✖ Pod default/example-cluster-storage-2 has an unready container: trace-log-forwarder
+Replace instances [stateless-5 storage-2 transaction-5] in cluster default/example-cluster [y/n]: y
+Error:
+found issues for cluster example-cluster. Please check them
```

Comment:
In the current version the error message shouldn't be printed when we use `auto-fix` to prevent confusion.